packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -66,6 +66,9 @@ export class DataStores implements IDisposable {
 
     private readonly disposeOnce = new Lazy<void>(() => this.contexts.dispose());
 
+    dataStoreCount: number;
```

Comment:
Can you add public qualifier for these? Also, these should be initialDataStoreCount or something like that. dataStoreCount indicates the current data store count which is incorrect. Also add comment for what it is.
Same for .